spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -331,6 +358,8 @@ object StandardizationJob {
     PerformanceMetricTools.addPerformanceMetricsToAtumMetadata(spark, "std", pathCfg.inputPath, pathCfg.outputPath,
       menasCredentials.username, cmd.cmdLineArgs.mkString(" "))
     standardizedDF.writeInfoFile(pathCfg.outputPath)
+
+    standardizedDF
```

Comment:
An addition to the above comment. Here you do
```
standardizedDF.write.parquet(pathCfg.outputPath)
```
and then return `standardizedDF`, and then in the parent method you do `standardizedDF.persist`

Alterantivalu you can do
``
standardizedDF.write.parquet(pathCfg.outputPath)
val outputDf = spark.read(pathCfg.outputPath)
outputDf
``
and then no need to do `outputDf.persist`.

Reply:
Done, as discusses above https://github.com/AbsaOSS/enceladus/pull/1332#discussion_r425937367